src/App.tsx

```diff
@@ -130,21 +135,19 @@ const App = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [appState])
 
-  // restore user and then fetch initial data
+  // update initial data when account is restored or app comes into foreground from background
   useEffect(() => {
-    if (!isRestored) {
-      dispatch(restoreUser())
-    } else {
+    if (
+      isRestored ||
+      (prevAppState === 'background' && appState === 'active')
```

Comment:
Why do we need to call `loadInitialData` even when `isRestored` is false? Is there a timing issue? That means they would be logged out but we still make the call right?

Reply:
I'm wondering if we even care about `isRestored` at all here. The only thing that is dependent on the account is notifications and I don't think we need to load them here anyway. I'll take a look at refactoring this...